src/main/java/hanieum/conik/adapter/company/persistence/CompanyEntity.java

```diff
@@ -0,0 +1,53 @@
+package hanieum.conik.adapter.company.persistence;
+
+import hanieum.conik.domain.company.CompanyStatus;
+import hanieum.conik.global.domain.BaseEntity;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "company")
+@Data
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class CompanyEntity extends BaseEntity {
```

Comment:
도메인 수준에서 사용하는 모델과 peresistence 레벨에서 사용하는 엔티티를 구분하지 않고 하나로 사용할거라면 Company로만 이름을 짓는게 좋을 것 같아요. BaseEntity 활용하는 거 너무 좋네요

```suggestion
public class Company extends BaseEntity
```